src/OmniSharp.Abstractions/Options/CodeActionOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OmniSharp.Options
+{
+    public class CodeActionOptions
+    {
+        public string[] LocationPaths { get; set; }
+
+        public IEnumerable<string> GetLocations(IOmniSharpEnvironment env)
+        {
+            if (LocationPaths == null) return Enumerable.Empty<string>();
+
+            var normalizePaths = new HashSet<string>();
+            foreach (var locationPath in LocationPaths)
+            {
+                if (Path.IsPathRooted(locationPath))
```

Comment:
If the paths are relative shouldn't they be rooted in the directory where the configuration file lives?  However this may be super painful to pull off (would require a custom options provider)